v0/destinations/singular/util.js

```diff
@@ -109,6 +110,10 @@ const platformWisePayloadGenerator = (message, isSessionEvent) => {
     throw new CustomError("[Singular] :: Platform name is missing", 400);
   }
   platform = platform.toLowerCase();
+  // checking if the os is one of ios, ipados, watchos, tvos
+  if (isAppleFamily(platform)) {
+    message.context.os.name = "iOS";
```

Comment:
We can directly update the platform to "IOS".

Reply:
but platform is not being used to set in the mapping right, rather is `context.os.name`

Reply:
Yes but along with that we can update platform as well. No need to update supported_platform in /config.js, as we are doing the same mapping as `isAppleFamily`.

Reply:
addressed